### ISSUE TITLE
Node Tree section should probably include Attr

### ISSUE THREAD

**Issue by cdumez** [whatwg/dom#457]
The "Node Tree" section [1] should probably include Attr now that it is a Node again, no?

[1] https://dom.spec.whatwg.org/#node-trees

**Comment by annevk** [comment]
Yeah, it seems there's some mismatches there as Attr objects do identify as nodes.

**Comment by annevk** [comment]
This seems like a bigger problem actually. There's numerous methods that do not consider what should happen when an `Attr` node is passed.

**Comment by annevk** [comment]
This got addressed bit-by-bit over the years. We're in pretty good shape now I think, including the section mentioned in OP.
